components/heading/src/Heading.js

```diff
@@ -60,7 +61,12 @@ Heading.propTypes = {
 	/**
 	 * The visual size of the heading
 	 */
-	size: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9]).isRequired,
+	size: PropTypes.oneOf([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]).isRequired,
```

Comment:
This is false because when `fontType` is `"brandFont"` it does not take the last couple of numbers